examples/chart/teleport-kube-agent/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: teleport-kube-agent
 apiVersion: v2
-version: "8.0.0-alpha.1"
-appVersion: "8.0.0-alpha.1"
+version: "8.1.0"
+appVersion: "8.1.0"
```

Comment:
@webvictim do we update this as part of a release? If now how do we keep the chart version in sync with the Teleport version?

Reply:
The Helm chart version gets updated as part of the `version.mk` Makefile: https://github.com/gravitational/teleport/blob/master/version.mk#L33-L45

Reply:
Ah that explains why `8.0.0-alpha.1` was still there and not `8.1.0` or `8.1.1-beta.1` as I was expecting. `master` still has `8.0.0-alpha.1` as its chart version, and that's what I started from.

I will undo this change then 👍